examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -1,6 +1,17 @@
 // This IDL was generated automatically by ZAP.
 // It is for view/code review purposes only.
 
+struct SemanticTagStruct {
```

Comment:
Are these structures shared across multiple clusters? I am wondering why they have no clusterid (they show up as global in .matter). Would it make more sense to attach these to specific cluster(s)?

Reply:
I thought they were missing cluster entries (like enums) however that does not seem to be the case. Resolving.